db.go

```diff
@@ -189,6 +192,12 @@ func Open(path string, mode os.FileMode, options *Options) (*DB, error) {
 	db.MaxBatchDelay = common.DefaultMaxBatchDelay
 	db.AllocSize = common.DefaultAllocSize
 
+	if options.Logger == nil {
+		db.Logger = getLogger()
```

Comment:
I have strong objection to have this library to output to it's own logger. A library should not output logs.